plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -88,11 +86,13 @@ protected boolean supportsCommentOnColumn()
         return false;
     }
 
+    protected abstract TestingMySqlServer getMysqlServer();
```

Comment:
It would be more felxible to require subclass to provide SqlExecutor, not the server instance.

Reply:
true I may change that.

Reply:
I also added:
```java
    @Override
    protected String getMysqlDatabase()
    {
        return mysqlServer.getDatabaseName();
    }
```
One of the test methods needed htat.